docs/decisions/0003-consolidation-into-frontend-platform.rst

```diff
@@ -0,0 +1,71 @@
+Consolidation of libraries into frontend-platform
+=================================================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+The frontend-platform repository replaces five earlier repositories:
+
+- edx/frontend-analytics
+- edx/frontend-auth
+- edx/frontend-base
+- edx/frontend-i18n
+- edx/frontend-logging
+
+We found that development across these five libraries was growing very difficult in a number of ways.
+
+Dependency graph complexity
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The overhead involved in maintaining individual packages and semantic version numbers was very difficult to reason and maintain.  Updates to low-level packages tended to cause cascading updates through the dependency tree.  If there were any breaking changes or incompatibilities present in the tree, an update might force dependent packages to absorb unrelated work, decreasing developer velocity and destabilizing our applications.
```

Comment:
'an update might force dependent packages to absorb unrelated work' This is still true no?

Reply:
True - I think I'll modify that part to point out that it can be hard to know when that actually happens.

Reply:
`s/difficult to reason/difficult to reason about/` ?